libcontainer/nsenter/nsexec.c

```diff
@@ -548,8 +584,13 @@ void nsexec(void)
 		bail("missing cloneflags");
 
 	/* Pipe so we can tell the child when we've finished setting up. */
-	if (socketpair(AF_LOCAL, SOCK_STREAM, 0, syncpipe) < 0)
+	if (socketpair(AF_LOCAL, SOCK_STREAM, 0, parentpipe) < 0)
```

Comment:
The comment for function `sendpid` and `recvpid` using `AF_UNIX`.  Although `AF_LOCAL` and `AF_UNIX` are equal, I think it should be consistent.